go/chat/errors.go

```diff
@@ -39,7 +39,7 @@ func NewPermanentUnboxingError(inner error) UnboxingError {
 type PermanentUnboxingError struct{ inner error }
 
 func (e PermanentUnboxingError) Error() string {
-	return fmt.Sprintf("error unboxing chat message: %s", e.inner.Error())
```

Comment:
don't think this should make it's way up to the user

Reply:
Hmm, why? Maybe just change to "decrypting" instead of "unboxing"? How about "error decrypting message: %s"?

Reply:
yeah i can go with that